test_runner/regress/test_timeline_detach_ancestor.py

```diff
@@ -50,15 +52,16 @@ def all() -> List["Branchpoint"]:
 
 @pytest.mark.parametrize("branchpoint", Branchpoint.all())
 @pytest.mark.parametrize("restart_after", [True, False])
+@pytest.mark.parametrize("write_to_branch_first", [True, False])
```

Comment:
Now there is just one test case which does both parametrization on restart_after and write_to_branch_first, I did test it locally with all tests having all of these but it feels like too much waste.

Of course, small tests are much better on CI runners because they spread out.